hoodie-common/src/main/java/com/uber/hoodie/common/model/CompactionOperation.java

```diff
@@ -83,7 +80,24 @@ public String getPartitionPath() {
     return partitionPath;
   }
 
-  public Map<String, Object> getMetrics() {
+  public Map<String, Long> getMetrics() {
```

Comment:
why change this ? we want to allow the flexibility to store complex metrics as well..

Reply:
Needed Avro compatible type here as they will be persisted in metadata.

Reply:
I ll let you two make the call. but if we are keep a single scalar type.. Can we do double, instead of long ?

Reply:
@n3nash:  As metrics are essentially numeric types, I have made changes to make metrics store double instead of long. If there is any future use-case to store tags like data, we can introduce them when such case arises. Let me know if this is ok with you ?